pkg/cloud/services/secretsmanager/secret.go

```diff
@@ -17,31 +17,33 @@ limitations under the License.
 package secretsmanager
 
 import (
-	"strings"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/uuid"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/secretsmanager"
-	apirand "k8s.io/apimachinery/pkg/util/rand"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
 	infrav1 "sigs.k8s.io/cluster-api-provider-aws/api/v1alpha3"
 	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/awserrors"
 	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/converters"
 	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/scope"
 )
 
 const (
-	parameterPrefix  = "aws.cluster.x-k8s.io"
-	namespacesPrefix = "namespaces"
-	clustersPrefix   = "clusters"
-	instancesPrefix  = "instances"
+	secretTypePrefix = "aws.cluster.x-k8s.io" // nolint
```

Comment:
Would it be better to rename this, or alternatively scope down the `// nolint` to `// nolint:gosec`?

Reply:
Changed the var